data-migrator/examples/entity-interceptor/README.md

```diff
@@ -0,0 +1,181 @@
+# Entity Interceptor Example
+
+This example demonstrates how to create an entity interceptor for the Data Migrator that can be packaged as a JAR and configured via YAML, without requiring Spring Boot annotations.
+
+## Overview
+
+The entity interceptor system allows you to:
+- Customize the conversion of Camunda 7 historic entities to Camunda 8 database models
+- Package entity interceptors in standalone JARs without Spring Boot dependencies
+- Drop JARs in the `configuration/userlib` folder of the Data Migrator distribution
+- Configure interceptors via `application.yml` instead of using `@Component` annotations
+- Set custom properties for each interceptor instance
+- Restrict interceptors to specific entity types (or handle all types)
+
+## Building the Interceptor
+
+```bash
+mvn clean package
+```
+
+This creates a JAR file in the `target` directory that can be deployed to the Data Migrator.
+
+## Deployment
+
+1. Copy the generated JAR file to the `configuration/userlib` folder of your Data Migrator installation
+2. Configure the interceptor in your `application.yml` file (see configuration example below)
+3. Start the Data Migrator with history migration mode: `./start.sh --history`
+
+## Configuration Example
+
+Add the following to your `application.yml`:
```

Comment:
Should we clarify we mean the data-migration application.yml file here (and where to find it)?